docker-compose.yml

```diff
@@ -21,6 +21,8 @@ services:
       - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-test}
       - AWS_ENDPOINT_OVERRIDE=http://localstack:4566/
       - AWS_S3_BUCKET_FAILED_FILES=data-store-failed-files-dev
+      - AWS_S3_BUCKET_FIND_DATA_FILES=data-store-find-data
+      - NOTIFY_FIND_API_KEY=${NOTIFY_FIND_API_KEY:?err}
```

Comment:
Think this should be commented out, with a note to uncomment if you want to test download, else it will crash on start-up if this is not set.